foundationdb/datadog_checks/foundationdb/check.py

```diff
@@ -52,6 +52,102 @@ def check(self, _):
 
         self.check_metrics(data)
 
+    def report_process(self, process):
+        if not process["address"]:
+            return
+        tags = [ "process:" + process["address"] ]
+
+        if "locality" in process:
+            locality = process["locality"]
+            if "machineid" in locality:
+                tags.append("machineid:" + locality["machineid"])
+            if "processid" in locality:
+                tags.append("processid:" + locality["processid"])
+            if "zoneid" in locality:
+                tags.append("zoneid" + locality["zoneid"])
```

Comment:
I think these are too variable to report

Reply:
(I asked Datadog about this case, and they pretty much said not to report anything that's hash-ish because tags shouldn't have such a large value space, and I suspect the process ID is recreated quite often.)